File: BillingAndSubscriptionSystem/EntityFrameWork/BillingAndSubscriptionSystem.Entities/Entities/Payment.cs
using BillingAndSubscriptionSystem.Entities.Enums;

namespace BillingAndSubscriptionSystem.Entities.Entities
{
    public class Payment
    {
        public int Id { get; set; }
        public decimal Amount { get; set; }
        public DateTime PaymentDate { get; set; }
        public PaymentStatus PaymentStatus { get; set; }
        public int SubscriptionId { get; set; }
    }
}

File: authProject/src/app/models/payment.ts
export interface IPaymentPayload {
  subscriptionId: number;
  amount: number;
  paymentDate: string;
  paymentStatus: number;
}

export interface IPaymentMessage {
  message: string;
}

export interface IOverduePayments {
  id: string;
  subscriptionId: number;
  amount: number;
  paymentDate: string;
  paymentStatus: number;
}
